pulp_rpm/app/tasks/comps.py

```diff
@@ -42,7 +45,9 @@ def parse_comps_components(comps_file):
     if comps.langpacks:
         langpack_dict = PackageLangpacks.libcomps_to_dict(comps.langpacks)
         langpack, created = PackageLangpacks.objects.get_or_create(
-            matches=strdict_to_dict(comps.langpacks), digest=dict_digest(langpack_dict)
+            matches=strdict_to_dict(comps.langpacks),
+            digest=dict_digest(langpack_dict),
+            pulp_domain=curr_domain,
```

Comment:
I thought the field was named `_pulp_domain`?

Reply:
Indeed.